docs/roadmap.md

```diff
@@ -32,6 +33,14 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
 
 #### Observability
 
+- [ ] Logging console enhancement
+- [ ] Monitoring stack upgrade including Prometheus, Prometheus Operator, Node exporter, kube-state-metrics etc.
+- [ ] Custom metrics support including application custom metrics dashboard, custom metrics HPA
+- [ ] Integration with Alertmanager
```

Comment:
Any more information or reference regarding Alertmanager? Do we have enhancement for this integration? @benjaminhuo